### PR TITLE
App Assurance Program

### PR DESCRIPTION
**Pull request by jhadobe**
## Purpose of this pull request

This PR adds information around the App Assurance Program that provides a higher level of review for Apps and Extensions.

Link to staging site: https://developer-stage.adobe.com/commerce/marketplace/guides/sellers/assurance/

### CODE REVIEW

**Comment by jhadobe** [comment]
I need to confirm the name of the program. If it applies to extensions and apps, then is it the Listing Assurance Program?

**Comment by glo75158** [comment]
Looks good

File: src/pages/guides/sellers/assurance.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+- Validate the change level against the level submitted by the developer
+- Check dependencies for any known vulnerabilities
+- Confirm that the listing is compatible with the newest version of Adobe Commerce
+
+The security code review consists of the following:
+
+- Check for outdated frameworks with known vulnerabilities
+- Check for possible SQL injection and data sanitization issues
+- Check databases for proper structure
+- Ensure proper uses of classes, methods, etc
```
Comment (keharper) [inline src/pages/guides/sellers/assurance.md:36]:
```suggestion
- Ensure proper usage of classes and methods
```

etc requires a period, and that wouldn't be parallel, so....

File: src/pages/guides/sellers/assurance.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: App Assurance Program
+description: This page describes the App Assurance Program for marketplace apps and extensions for Adobe Commerce.
```
Comment (keharper) [inline src/pages/guides/sellers/assurance.md:3]:
```suggestion
description: Learn about the App Assurance Program for Marketplace apps and extensions for Adobe Commerce.
```

**Review by keharper — APPROVED** [review]

**Pull request merged**